README.md

```diff
@@ -1,9 +1,17 @@
-# ECE 464: Databases
```

Comment:
do you want to make these changes in the main README or a separate directory? i think having a main README is useful